IPython/html/static/style/style.min.css

```diff
@@ -8964,6 +9001,13 @@ ul#new-menu {
     header */
   margin-bottom: -1px;
 }
+.header_edit {
+  flex: 0 0 auto;
```

Comment:
it looks like all of the headers and sites have the same flex value. That would suggest this CSS should move to `#header` and `#site` instead of being copied to every instance. Is there a reason these should all be treated separately?